### ISSUE TITLE
New error code and condition, I202

### ISSUE THREAD

**Issue by pgjones** [PyCQA/flake8-import-order#107]
As newlines represent a change of import grouping, additional newlines should be considered incorrect. For example,
```python
import asyncio

import secrets
```
Should raise a I202 as the newline does not represent an acceptable newline location (as the grouping is the same).

**Comment by mitar** [comment]
Yes, this surprised me as well. One can add random newlines and it does not complain.
